build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
+}
+
+repositories {
+    maven {
+        url = 'http://artifacts.openmicroscopy.org/artifactory/maven'
+    }
+    maven {
+        url = 'http://repo.maven.apache.org/maven2'
+    }
+}
+
+dependencies {
+    implementation 'org.openmicroscopy:omero-server:5.5.6'
+    implementation 'com.google.guava:guava:27.1-jre'
+    implementation 'io.vertx:vertx-core:3.9.0'
+    implementation 'io.vertx:vertx-web:3.9.0'
+}
+
+group = 'org.openmicroscopy'
+version = '0.1.0-SNAPSHOT'
+description = 'omero-ms-zarr'
+sourceCompatibility = '1.8'
```

Comment:
java {
    sourceCompatibility = JavaVersion.VERSION_1_8
}

Reply:
What does this do?

Reply:
it indicates that it is for the java plugin

Reply:
Done